Services/PriceVariantProvider.cs

```diff
@@ -40,11 +40,12 @@ public async Task AddPrices(IList<ShoppingCartItem> items)
                         {
                             var attributesRestrictedToPredefinedValues = _predefinedValuesService
                                 .GetProductAttributesRestrictedToPredefinedValues(product.ContentItem)
-                                .Select(attr => attr.PartName + "." + attr.Name)
+                                .Select(attr => $"{attr.PartName}.{attr.Name}")
```

Comment:
The concatenation was actually much better.

Reply:
You're right